### ISSUE TITLE
[ACE] ERROR - Extension ace_medical.dll not installed

### ISSUE THREAD

**Issue by JediNick** [acemod/ACE3#1679]
ACE3 Version: 3.1.1
STABLE

**Mods**
* @cba_a3
* @ace3

**Placed ACE3 Modules:**
None
**Description:**
Whenever I load into any mission or even editor I load into the game and this error pops up and allows me to click OK then boots me to mission screen

**Steps to reproduce:**
Start Arma 3 choose any mission or load into editor
**Where did the issue occur?**
Both Multiplayer and singleplayer

OTHER
http://i.imgur.com/FqSqk6h.jpg

**Comment by Glowbal** [comment]
You need the `ace_medical.dll` file in your ACE folder - where it is located by default with the download.

**Comment by JediNick** [comment]
it is in my folder however the issue persists

**Comment by JediNick** [comment]
Reinstalling has also been attempted with no success

**Comment by Glowbal** [comment]
Could you provide an rtp log file, please?

**Comment by JediNick** [comment]
How can I get a rtp log? I'm quite new to this

**Comment by JediNick** [comment]
Note: I am using mods for that RTP log but the issue also happens with only CBA and ACE

**Comment by Glowbal** [comment]
That is the output of our debug button. I'd like a peak at the rtp log file that is created by the arma game itself. 

https://community.bistudio.com/wiki/Crash_Files#Arma_3
`%userprofile%\AppData\Local\Arma 3`
If you look in that folder, you will find your rtp files. If there are non present, please run without `-noLogs`, replicate the error and check again for the rtp files.

Copying the rtp file contents to a [pastebin](http://pastebin.com) would be fine.

**Comment by JediNick** [comment]
Oops I posted a debug

**Comment by JediNick** [comment]
http://pastebin.com/dWspTvuV

**Comment by JediNick** [comment]
That the right thing?

**Comment by PabstMirror** [comment]
```
16:38:08 [ACE]: ACE is version 3.1.1.1.
16:38:08 [ACE] Extension version: ace_break_line: 3.0.1-9f3a57b
16:38:08 [ACE] ERROR: Extension ace_medical.dll not installed.
````
This is weird.   Could you completely delete your `@ace` folder and then re-extract from the current release: https://github.com/acemod/ACE3/releases/tag/v3.1.1

**Comment by JediNick** [comment]
Tried this, same issue

**Comment by JediNick** [comment]
However when another error message popped up it cancelled the ACE error allowing me to play without issue but the error will reappear every time I rejoin a server

**Comment by ztrimble** [comment]
Just wanted to say friend is having exact same issue, no solution as of now.

**Comment by Glowbal** [comment]
@JediNick thank you for the rtp log.

What is your OS?

**Comment by JediNick** [comment]
My OS is windows 7

**Comment by ztrimble** [comment]
Also we tried a few things that did not work:

Turning off antivirus
Reinstalling ACE
Verifying game cache
Rolling back ace to the last version
Loading only ace and cba

**Comment by Half-Health** [comment]
Hey there Glowbal, I am ztrimble's friend that was having this error as well. Here is my information:

CBA Version - http://i.imgur.com/rPA4VF1.png
ACE Version - http://i.imgur.com/sE0H5Z4.png
PC Info - http://i.imgur.com/GUxeAR1.png
RPT - http://pastebin.com/U2cyhYGN

Video of whats going on (sorry for the awkwardness):
https://www.youtube.com/watch?v=weZ4_maEAss

**Comment by JediNick** [comment]
That video shows what happens in my game aswell

**Comment by sargken** [comment]
Make sure the dll isnt being blocked, properties and sometimes it will say unblock.

**Comment by JediNick** [comment]
@sargken  Dont think this would be the issue as I tried running arma 3 as administrator and had the same issue

**Comment by Glowbal** [comment]
@ztrimble 
>  Rolling back ace to the last version
Which version did you roll back to? The previous release version should not pop-up with the extension error.

-
Could someone who has this issue try to replace the `ace_medical.dll` in your ace folder by this: https://github.com/acemod/ACE3/blob/master/ace_medical.dll?raw=true

**Comment by Half-Health** [comment]
When I reverted back, it was to version 3.0.1 which worked fine. I also just tried that new ace_medical file, same issue.

**Comment by JediNick** [comment]
It's a odd bug, if you manage to get in game the medical system still works fine but it's seems it's believing that the medical dll is missing and kicks you out for it

**Comment by esteldunedain** [comment]
> It's a odd bug, if you manage to get in game the medical system still works fine but it's seems it's believing that the medical dll is missing and kicks you out for it

ACE is designed to work without the dll, so it's not weird. The dll offers better performance, so we require it on the clients.

To be honest, I tend to think the problem is related to anti-viruses blocking that dll from loading, probably due to it's name. Could you inform what anti-virus, anti-spyware softwares you guys are using?

**Comment by Half-Health** [comment]
I'm using AVG, I even made an exception specifically for the file, but to no avail. I also tried turning my anti-virus off and that also failed to make a difference.

**Comment by Glowbal** [comment]
I will make it only warn about the not installed dll.

**Comment by Half-Health** [comment]
As an update.. I have no idea what changed, but it works for me now. Nothing on my PC should have changed between yesterday and today, but ArmA is working fine for me and not giving me the ace_medical error anymore.

**Comment by gienkov** [comment]
Can you try launching game through Arma 3 Sync? It solved this issue buddy
from my community had.

**Comment by JediNick** [comment]
@Glowbal  The file you asked for me to use did not work, same issue


"Could someone who has this issue try to replace the ace_medical.dll in your ace folder by this: https://github.com/acemod/ACE3/blob/master/ace_medical.dll?raw=true"

**Comment by ViperMaul** [comment]
Has AVG been notified about this?

**Comment by Glowbal** [comment]
Is everyone who has this issue using AVG?

**Comment by Half-Health** [comment]
I am using AVG, however, the issue has gone away.

**Comment by bux578** [comment]
> the issue has gone away.

**Comment by xJenny99x** [comment]
I still have this issue.

**Comment by PabstMirror** [comment]
From #1926
>Okay figured it out, you need to launch the arma 3 launcher first for battleye to update itself, if you launch it through command lines it doesn't give a fuck and doesn't update I'll add this to the wiki beause this is a very important thing to know.

Please also try fully deleting @ACE folder and reinstalling 3.2.1

Can also try updating battleeye manually: http://www.battleye.com/downloads/

**Comment by PabstMirror** [comment]
If still having problems make a new issue, but most have gotten it working after trying these steps.

**Comment by Tress16** [comment]
I have the same issue with the clipboard.dll. But I think only if I'm the host. I'm using the newest versions of cba, ace3, bwmod, rhs:russia and rhs:usa and the McAfee Security. Joining the game isn't possible. If I run Ace local on 3den, it works.

**Comment by bux578** [comment]
@Tress16 please don't revive closed issues. Open a new one with a proper description: http://ace3mod.com/wiki/user/how-to-report-an-issue.html
